leaflet-treering.js

```diff
@@ -34,6 +34,7 @@ function LTreering (viewer, basePath, options) {
     this.meta.ppm = options.initialData.ppm;
   }
 
+  //add window.name.includes('popout') to have error only appear in measurement mode
```

Comment:
I'm not clear what this comment refers to - a note for a future change?

Reply:
Future change (Issue #24) I found while mucking around.